src/python/pants/backend/python/goals/pytest_runner.py

```diff
@@ -239,13 +239,26 @@ async def setup_pytest_for_target(
     )
     pytest_runner_pex, config_files = await MultiGet(pytest_runner_pex_get, config_files_get)
 
+    # The coverage and pytest config may live in the same config file (e.g., setup.cfg, tox.ini
+    # or pyproject.toml), and wee may have rewritten those files to augment the coverage config,
```

Comment:
```suggestion
    # or pyproject.toml), and we may have rewritten those files to augment the coverage config,
```

Reply:
I'll save a tree and fix this another time.